server/controllers/onboardingToolController.js

```diff
@@ -145,12 +145,14 @@ async function handleOnboardingToolUse({
     };
     const { id } = await insertShip(ship);
     console.log("Inserted ship", id);
-    if (mode === 'paid') {
-      const profile = await getUserProfile(userId);
-      const { available_ships } = profile; // current
-      const profilePayload = { available_ships: available_ships - 1 }; // updated
-      await updateUserProfile(userId, profilePayload);
-    }
+
+    // removing deduct available ships from here.
```

Comment:
you can remove the unused code instead of committing, rest looks good

Reply:
Removed commented code. Please review now.